src/SeqViz/handlers/selection.tsx

```diff
@@ -34,21 +37,28 @@ export const defaultSelection = {
 export const SelectionContext = React.createContext(defaultSelection);
 SelectionContext.displayName = "SelectionContext";
 
+export type SeqVizMouseEvent = React.MouseEvent & {
+  type: string;
+  clientX: number;
+  clientY: number;
```

Comment:
this are included already, no? https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent